### ISSUE TITLE
(OIHM) Configuration Questions, whether in a topic or not, exhibit odd behaviour as to whether they require (for input) certain ranges.

### ISSUE THREAD

**Issue by celiapetty** [dbrownukk/EFD_v2#613]
Instance QA
App: OIHM
Study: New Cattle. 
Project: Highland

New Topic 'Forestry' created in Topic module under OIHM Extra menu and questions added-see OIHM extra
Questions import as expected
However, I haven't been able to remove ranges which appear in all 3 questions.
I will also need to explain LOV (have looked this up....) 

We will be using this enhancement in the next Training (Foundation II) scheduled for July 2021.

**Comment by daiClegg** [comment]
Questions, whether in a topic or not, exhibit odd behaviour as to whether they require (for input) certain ranges.
Will investigate

**Workaround:** Change the scope to Study & or switch question type between Text/Decimal Range/LoV. Eventually, stray ranges will disappear. I have already done this (as part of my testing) for the Forestry Topic.

I have not yet tested whether the answers for such questions are correctly validated when added to a Study.

**Comment by daiClegg** [comment]
I cleaned out the old test questions in Testing and tried many edits: change of type/in-topic/standalone and i can't reproduce any aberrant behaviour.
it may be related to creating new questions (blocked by #613)
will test further when that is fixed

meanwhile will try on QA instance

**Comment by daiClegg** [comment]
I notice that it is possible to create new Configuration Questions in the context of a Topic.

**Comment by dbrownukk** [comment]
Was a bug, iterator issue when checking if existing Topic already used. 

Fixed. Deployed to IDAPS_Livelihoods_TESTING

**Comment by daiClegg** [comment]
I did more testing on configuration questions and i cannot find any problems, so closing this.
